### DISCUSSION TITLE
Disable the printing button

### DISCUSSION THREAD

**Discussion by lukaszrudnicki** [outline/outline#5832] [Self Hosting]
Hello, first of all, I would like to point out that I am a beginner, so the problem may lie in the lack of basics.

I would like to disable the printing option for everyone. I downloaded the application and found:

```
export const printDocument = createAction({
name: ({ t, isContextMenu }) =>
isContextMenu ? t("Print") : t("Print document"),
analyticsName: "Print document",
section: DocumentSection,
icon: <PrintIcon />,
visible: ({ activeDocumentId }) => !!(activeDocumentId && window.print),
perform: async () => {
queueMicrotask(window.print);
},
});
```

I changed:

`visible: ({ activeDocumentId }) => false,`

I created an image and installed it, but I still have the "print" button.

I assume that I need to change something else somewhere, but I don't even have an idea where to look for it.

**Comment by tommoor** [comment]
That should do it, this is only the print button in the document menu of course. You cannot disable browser print generally
